src/frontend/web_application/src/scenes/Discussion/components/MailMessage/index.jsx

```diff
@@ -57,16 +58,11 @@ class MailMessage extends Component {
 
   renderBody = () => {
     const { message, isLocked, encryptionStatus } = this.props;
+    const body = message.body_is_plain ? `<pre>${anchorme(message.body, { emails: false, attributes: [{ name: 'target', value: '_blank' }] })}</pre>` : message.body;
 
     if (!isLocked) {
-      if (!message.body_is_plain) {
-        return (
-          <TextBlock nowrap={false} className="s-mail-message__content" dangerouslySetInnerHTML={{ __html: message.body }} />
-        );
-      }
-
       return (
-        <TextBlock nowrap={false}><pre className="s-mail-message__content">{message.body}</pre></TextBlock>
+        <TextBlock nowrap={false} className="s-mail-message__content" dangerouslySetInnerHTML={{ __html: body }} />
```

Comment:
actually yes, I think it's too dangerous

Reply:
try to receive a plain text message w/ the following and see what happens:

```
<script>alert(1)</script>
```

Reply:
Yes I understand the problem.